frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -236,6 +236,14 @@ const TaipyPlotlyButtons: ModeBarButtonAny[] = [
     },
 ];
 
+const updateArrays = (sel: number[][], val: number[], idx: number) => {
+    if (idx >= sel.length || val.length !== sel[idx].length || val.some((v, i) => sel[idx][i] != v)) {
+        sel = sel.concat();
```

Comment:
Please add // Shallow copy
That concat() syntax is really misleading.

Reply:
I could write sel = [...sel] if you's rather ?

Reply:
With the comment (and why don't you keep the capital 'S', I don't know), everything is clear! Thanks.